ql/src/test/queries/clientpositive/jdbc_handler.q

```diff
@@ -1,5 +1,4 @@
 --! qt:dataset:src
---! qt:disabled:flaky HIVE-23709
```

Comment:
We have no proof that this is not flaky anymore so leave it disabled for the time-being and add new tests if necessary.

Reply:
Done,. Leave it disabled.